.github/automation/.drone.yml

```diff
@@ -29,7 +29,7 @@ steps:
   - export DEBIAN_FRONTEND=noninteractive
   - apt-get update && apt-get install -y git build-essential cmake scons gcc-10 g++-10
   - update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 1 && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 1
-  - .github/automation/build_acl.sh  --version 23.02.1 --arch armv8a --multi_isa --root-dir $(pwd)/ComputeLibrary
+  - .github/automation/build_acl.sh  --version 23.11 --arch armv8a --multi_isa --fixed_format_kernels --root-dir $(pwd)/ComputeLibrary
```

Comment:
Is the `--fixed_format_kernels` needed or related to this change? As far as I can see, this isn't an option supported in `build_acl.sh` and will cause it to fail. (Comment applies to line 86 and 117 too)

Reply:
thanks for the review, @jondea . you're right,  looks like it's not an option to pass, I will update the build_acl.sh to use the right flag for fixed_format_kernels there.